webpack/webpack-embedded-dependencies-plugin/src/EmbeddedDependenciesWebpackPlugin.ts

```diff
@@ -328,17 +323,18 @@ export default class EmbeddedDependenciesWebpackPlugin implements WebpackPluginI
     const files: InputFileSystemReadDirResults = await LegacyAdapters.convertCallbackToPromise(
       inputFileSystem.readdir,
       modulePath,
-      { withFileTypes: true }
+      { withFileTypes: true, encoding: 'buffer' }
```

Comment:
If `encoding` is a required parameter now, pass `utf8` or `utf-8` so that the returned type of `file.name` is `string`.